src/components/Footer.jsx

```diff
@@ -57,9 +85,43 @@ class SiteFooter extends React.Component {
             <img
               style={{ maxHeight: 45 }}
               src={logo || config.LOGO_TRADEMARK_URL}
-              alt={intl.formatMessage(messages['footer.logo.altText'])}
+              alt={getConfig().FOOTER_LOGO_ALT_TEXT || intl.formatMessage(messages['footer.logo.altText'])}
             />
           </a>
+          )}
+          <div className="copyright-col">
+            {getConfig().TRADEMARK_TEXT
+            && (
+            <div className="trademark-text text-gray-500 small">
```

Comment:
I still think `text-gray-500` is too specific to be hard-coded.  Maybe remove it and reimplement it in the CSS, so it can be overriden more explicitly by theming later?

Reply:
This was a bootstrap mixin. I have updated it now.